src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/caseofficer/CaseOfficerRespondentEvidenceSubmittedPersonalisation.java

```diff
@@ -67,6 +74,86 @@ public String getReferenceId(Long caseId) {
                 .put("appellantGivenNames", asylumCase.read(APPELLANT_GIVEN_NAMES, String.class).orElse(""))
                 .put("appellantFamilyName", asylumCase.read(APPELLANT_FAMILY_NAME, String.class).orElse(""))
                 .put("linkToOnlineService", iaExUiFrontendUrl)
+                .put("legalRepCompany",   asylumCase.read(LEGAL_REP_COMPANY, String.class).orElse(""))
+                .put("legalRepCompanyAddress", formatCompanyAddress(asylumCase))
+                .put("legalRepName",   asylumCase.read(LEGAL_REP_NAME, String.class).orElse(""))
+                .put("legalRepEmail",   asylumCase.read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class).orElse(""))
+                .put("legalRepReference",   asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class).orElse(""))
                 .build();
     }
+
+    public String formatCompanyAddress(AsylumCase asylumCase) {
```

Comment:
Should this be made private?

Reply:
I have made one utility class where have place this method, because this method is used by two classes.
Hope that should be fine now.